windows-agent/internal/proservices/landscape/executor.go

```diff
@@ -141,10 +145,6 @@ func (e executor) install(ctx context.Context, cmd *landscapeapi.Command_Install
 		return fmt.Errorf("skipped installation: %v", err)
 	}
 
-	if err := gowsl.Install(ctx, distro.Name()); err != nil {
```

Comment:
stupid question, probably hidden by the diff, but where is the distro install code now when there is no rootfs url provided? Was that code duplicated here?

Reply:
I suspect that's a leftover:

Looking at main: inside that method we have such call, and below it we have a check whether we have a URL or store ID, based on which we call installFromURL or installFromMicrosoftStore: https://github.com/canonical/ubuntu-pro-for-wsl/blob/main/windows-agent/internal/proservices/landscape/executor.go#L159-L167

Inside the later, we have that exact same line:

https://github.com/canonical/ubuntu-pro-for-wsl/blob/main/windows-agent/internal/proservices/landscape/executor.go#L226

We don't want to unconditionally install from the Store without the URL validation.

Reply:
Indeed, that was clearly a leftover. Good riddance :)